### PR TITLE
Update TRUSTZONE profile

### PR DESCRIPTION
**Pull request by ChristianBrandenburg**
Hi Paul 

We would like to update our profile with the following content and pictures. 

Kind regards
Christian Brandenburg
Sales Engineer @ TRUSTZONE

### CODE REVIEW

**Review by vanbroup — CHANGES_REQUESTED** [review]
With the mentioned changes the profile would look like:

image

Can you crop the profile photos to be more like a square, this would give all images an equal with, like other profiles?

To locally preview the site, install/download the [hugo command](https://gohugo.io/getting-started/installing/) and run `hugo server -D` in the root of the repository.

You can also include a video about TRUSTZONE in the profile body if you like, include the following snippet in the `trustzone.md` file and change to code to the video id you would like to include.

`{{< youtube jdpiTcc9d4U >}}`

File: data/members/trustzone.yaml [hunk 1]
```diff
@@ -49,9 +48,18 @@ social:
...
     name: Jacob Højmark Nielsen
-    role: CEO
+    role: Chief Executive Officer
     social:
       twitter: 
       linkedin: https://www.linkedin.com/in/jnielsen4/
     description:
-      Jacob is the CEO and one of the founders of TRUSTZONE, he represents the organization at the PKI Consortium.
\ No newline at end of file
+      Jacob is the CEO and Co-founder of TRUSTZONE and represents the organization at the PKI Consortium.
```
[the remark this answers is not in the document]
Comment (ChristianBrandenburg) [reply data/members/trustzone.yaml]: Yup, I'll ask Jacob to add some more backstory

File: data/members/trustzone.yaml [hunk 2]
```diff
@@ -49,9 +48,18 @@ social:
...
+
+  - id: stuven
+    name: Svein Tuven
+    role: Senior Vetting Officer
+    social:
+      twitter:
+      linkedin: https://www.linkedin.com/in/svein-tuven-a522a815/
+    description:
```
[the remark this answers is not in the document]
Comment (ChristianBrandenburg) [reply data/members/trustzone.yaml:63]: Yup, I'll ask Svein to add some more backstory

File: data/members/trustzone.yaml [hunk 3]
```diff
@@ -49,9 +48,18 @@ social:
 representatives:
   - id: jhnielsen
```
[the remark this answers is not in the document]
Comment (ChristianBrandenburg) [reply data/members/trustzone.yaml:49]: Thank you for the explanation. I will rename the photo.

File: data/members/trustzone.yaml (hunk 1 above)
[the remark this answers is not in the document]
Comment (ChristianBrandenburg) [reply data/members/trustzone.yaml]: I have updated Jacobs background section

File: data/members/trustzone.yaml (hunk 2 above)
[the remark this answers is not in the document]
Comment (ChristianBrandenburg) [reply data/members/trustzone.yaml:64]: I have updated Sveins background section

File: data/members/trustzone.yaml [hunk 4]
```diff
@@ -19,7 +18,7 @@ workingGroups:
 # Blog posts
 blog: 
   url: https://trustzone.com/news/
-  feed: https://trustzone.com/feed/
+  feed:
```
[the remark this answers is not in the document]
Comment (ChristianBrandenburg) [reply data/members/trustzone.yaml]: We have decided to to keep the blog posts in after all

**Review by vanbroup — APPROVED** [review]

**Pull request merged**